hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieFileGroup.java

```diff
@@ -62,14 +59,16 @@
   /**
    * The last completed instant, that acts as a high watermark for all getters
    */
-  private final Optional<HoodieInstant> lastInstant;
+  private final com.google.common.base.Optional<HoodieInstant> lastInstant;
```

Comment:
who else uses Optional in this file? could nt we import guava optional and avoid explict package naming in definition?

Reply:
+1